frontend/src/store/wallet.ts

```diff
@@ -358,6 +358,11 @@ export default function useWalletStore() {
     setLastWallet(null);
   }
 
+  function setLanguage(language: string) {
+    if (!onboard.value) return;
+    onboard.value.state.actions.setLocale(language);
+  }
```

Comment:
Are there better ways to compose this functionality with the settings setLanguage function? Are composables able to share state with other composables?

Reply:
I'm not sure exactly what you mean by "composables", but I think what you're asking is how we can have this wallet.ts `setLanguage` method called as part of the settings.ts `setLanguage` method? I _think_ you might be able to just import this method in the settings store and call it there, or does vue given an error with that?

Reply:
Oh wow I did not realize this was an actual term in the composition API world haha https://vuejs.org/guide/reusability/composables.html

Reply:
When I tried calling useWalletStore in useSettings `setLanguage` I was getting a few errors. It seemed like there was a weird recursion, but I didn't dig too deep. So, I ended up moving away from it.

Reply:
This approach is probably fine for now, maybe just add a comment to the wallet `setLanguage` method documenting that it should never be called on it's own, and should always be called along with the settings.ts version

Reply:
Done in e7a01d1, so will merge